docs/src/rings.md

```diff
@@ -335,13 +340,14 @@ expression method below is preferred.
 
 ### Expressions
 
-Alternatively, one can implement a method
+To obtain best results when printing composed types derived from other types, e.g., polynomials,
+the following method should be implemented
```

Comment:
meta-comment: I really dislike that the ncring and ring documentation overlap so much, and duplicate tons of stuff. That just adds work, leads to drift, and also is worse for the user, who now has to go over everything with a comb to find the difference. Why not have a page which lists all the things that are the same; and then ring and ncring just say "everything from that page, plus the following few points" ? Or alternatively, keep the ncring page, and change the ring page to say "this is like ncring, except that the following are added: ... ; and the following are replaced by this and that ..."

Reply:
I couldn't agree more with that.

Reply:
I agree, but this is out of scope of this PR. Might be worthwhile to open an issue.

Reply:
Indeed it is outside the scope of this PR (that's what I meant to convey with "meta-comment", I'll make sure to write it explicitly in the future).

I've filed issue #725 about this.

Reply:
I now what you meant. I just wrote that this is orthogonal to this pull request and that you should open and issue if you don't want your comment to be forgotten. It has nothing to do with it being explicit or not.